ngsild-payloads/userProfile.jsonld

```diff
@@ -1,6 +1,10 @@
 {
     "id": "urn:ngsi-ld:UserProfile:romain.magnani@egm.io",
     "type": "UserProfile",
+    "userInfos": {
+        "type": "JsonProperty",
+        "value": {}
+    },
```

Comment:
why is the value empty ?

Reply:
Because I did this PR in my bed yesterday and was lazy ^^